httpserver/src/main/kotlin/tbdex/sdk/httpserver/TestServer.kt

```diff
@@ -0,0 +1,70 @@
+package tbdex.sdk.httpserver
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.response.respond
+import tbdex.sdk.httpclient.models.Exchange
+import tbdex.sdk.httpserver.models.FakeExchangesApi
+import tbdex.sdk.httpserver.models.FakeOfferingsApi
+import tbdex.sdk.protocol.models.Message
+import tbdex.sdk.protocol.models.Offering
+import web5.sdk.crypto.InMemoryKeyManager
+import web5.sdk.dids.methods.dht.DidDht
+
+// todo remove this before merging!!
+/**
+ * Main function to test the server.
+ */
+fun main() {
+  val serverConfig = TbdexHttpServerConfig(
+    port = 8080,
+    pfiDid = DidDht.create(InMemoryKeyManager()).uri,
+    offeringsApi = FakeOfferingsApi(),
+    exchangesApi = FakeExchangesApi()
+  )
+
+  val tbdexServer = TbdexHttpServer(serverConfig)
+
+  tbdexServer.createExchange { call, rfq, offering, replyTo ->
```

Comment:
fyi @chris-tbd this is how i imagine you could implement the server once this change is actually implemented. much nicer, imo, and hopefully solves the problem you meant to address when you wrote issue #151 

please let me know if u have any questions

Reply:
Cc @angiejones

Reply:
Love this way of handling it!

Reply:
now type checking is redundant 🎉 
![image](https://github.com/TBD54566975/tbdex-kt/assets/118510103/cf196616-4686-4c3e-a28f-ac787156cd9c)